### ISSUE TITLE
3.25.0 breaks aws_ebs_volume with size": only one of `size,snapshot_id` can be specified

### ISSUE THREAD

**Issue by dbgeek** [hashicorp/terraform-provider-aws#17235]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

```
terraform -v
Terraform v0.14.5
+ provider registry.terraform.io/hashicorp/aws v3.25.0
```

### Affected Resource(s)

* aws_ebs_volume 

### Terraform Configuration Files

````
terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 3.0"
    }
  }
  required_version = ">= 0.14"
}

resource "aws_ebs_volume" "ebs_volume" {
  availability_zone = "eu-west-1a"
  size              = 16
  type              = "gp3"
  snapshot_id       = "snap-0fb629cbb0446b995"
  encrypted         = true
}
```

### Actual Behavior

```
Error: ExactlyOne

  on main.tf line 13, in resource "aws_ebs_volume" "ebs_volume":
  13:   size              = 16

"size": only one of `size,snapshot_id` can be specified, but
`size,snapshot_id` were specified.


Error: ExactlyOne

  on main.tf line 15, in resource "aws_ebs_volume" "ebs_volume":
  15:   snapshot_id       = "snap-0fb629cbb0446b995"

"snapshot_id": only one of `size,snapshot_id` can be specified, but
`size,snapshot_id` were specified.
```

### Steps to Reproduce

try to create EBS volume that specifies both size and snapshotid
main.tf 
```
resource "aws_ebs_volume" "ebs_volume" {
  availability_zone = "eu-west-1a"
  size              = 16
  type              = "gp3"
  snapshot_id       = "snap-0fb629cbb0446b995"
  encrypted         = true
}
```

1. `terraform plan`

### References

This change is what breaks it
https://github.com/hashicorp/terraform-provider-aws/blob/master/aws/resource_aws_ebs_volume.go#L68

AWS API supports both size & snapshot. Size should just be equal to or larger than snapshot size
https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVolume.html

ping @ewbankkit

**Comment by ewbankkit** [comment]
@dbgeek Thanks for raising this issue.
Indeed, that does seem to be the case, which conflicts with the statement in the same paragraph 😄:

> You must specify either a snapshot ID or a volume size.

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 3.26.0 of the Terraform AWS Provider, on Thursday. 👍
